pkg/config/config.go

```diff
@@ -201,6 +211,24 @@ func validateConfig(cfg *Config) error {
 		}
 	}
 
+	var defaultEntries = []util.Entry{
```

Comment:
Is this a good place to add the defaults?

I was in doubt between here and inside the setuoLoggingConfig function

Reply:
why not inside the config struct.
https://github.com/tellor-io/telliot/blob/master/pkg/config/config.go#L99

I think we should even rename it to `defaultConfig`